students/csimmons/Lesson09/donor_models.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# Craig Simmons
+# Python 210
+# mailroom_oo.py assignment
+# Donor classes 
+# Created 1/17/2021 - csimmons
+
+from operator import itemgetter
+
+class Donor(object):
+       
+    def __init__(self, name='', donations= []):
+        self.name = name
+        self.donations = donations
+
+    def __str__(self):
+        return f'Donor Name is: {self.name}'
+
+    def __repr__(self):
+        return f'Donor({self.name})'
+        
+    @property
+    def total_donations(self):
+        return sum(self.donations)
+
+    @property
+    def number_donations(self):
+        return len(self.donations)
+        
+    @property
+    def avg_donation(self):
+        return sum(self.donations) / len(self.donations)
+
+
+class DonorCollection(object):
+
+    donors_db = {
+    'Mary Newcomer' : [10000, 2500, 300],
+    'Christine Ruotolo' : [3000, 6000, 750, 20000],
+    'Martin Acevedo' : [2000, 5000],
+    'Sutton Keaney' : [24500, 500, 3000, 5000, 1000],
+    'David Basilio' : [750, 750, 750, 750, 5000, 750, 750],
+    'Andrew Laughlin' : [2500, 500, 40000, 50],
+    'Hussein Saffouri' : [1000, 1000, 2100, 7000, 55000],
+    }
```

Comment:
this data should be on instance level not class level.

Use class attributes for things that should belong to a class, for example something like a configurable setting.

Instance attributes should be unique to each instance of the class, and in this case donors are unique to each instance.